paasta_itests/static_port.feature

```diff
@@ -1,25 +1,29 @@
 Feature: setting host_port in marathon.yaml creates a marathon app that behaves as we expect.
+  @skip
```

Comment:
whoa whoa whoa

These passed when I ran them manually when I did this PR.

Why are we skipping so many things?